zowe/zowe-cli
# Fix api for uploading buffer to uss file

**What It Does**
- LTS Breaking: Changed return type of `Upload.bufferToUssFile` to return `IZosFilesResponse` object instead of string
- BugFix: Fixed `Upload.bufferToUssFile` not normalizing new lines when uploading plain text

**How to Test**
Zowe CLI doesn't call this API so no easy way to test it there. However we should be able to test it in FileSystemProvider PR for Zowe Explorer 😋 https://github.com/zowe/zowe-explorer-vscode/pull/2777

**Review Checklist**
I certify that I have:


**Additional Comments**
